api/src/components/publicationVersion/service.ts

```diff
@@ -1732,12 +1728,9 @@ export const postPublishHook = async (
         await updateCanonicalDOI(publicationVersion.publication.doi, publicationVersion);
 
         // (Re)index publication in opensearch.
-        if (publicationVersion.versionNumber > 1 || forceReindex) {
-            // Delete old OpenSearch record.
-            await publicationService.deleteOpenSearchRecord(publicationVersion.versionOf);
-        }
-
-        await publicationService.createOpenSearchRecord({
+        // TODO: remove this extra logging if we don't observe ARI imports stalling here for some time.
+        console.log(`Indexing publication ${publicationVersion.versionOf} in opensearch.`);
```

Comment:
Guess you have a ticket to revisit this?

Reply:
We do now!